Assets/StreamingAssets/Data/Furniture.xml

```diff
@@ -277,6 +277,45 @@
     <LocalizationCode>furn_metal_smelter</LocalizationCode>
     <UnlocalizedDescription>furn_metal_smelter_desc</UnlocalizedDescription>
   </Furniture>
+	
+<Furniture objectType="Metal Furnace">
+    <Name>Metal Furnace</Name>
+    <Description>Smelts your raw ores into metal.</Description>
+    <TypeTag>Processor</TypeTag>
+    <MovementCost>1</MovementCost>
+    <PathfindingModifier>1.5</PathfindingModifier>
+    <PathfindingWeight>2</PathfindingWeight>
+    <Width>3</Width>
+    <Height>3</Height>
+
+    <BuildingJob jobTime="0" />
+
+    <JobSpotOffset X="1" Y="0" />
+
+	<FactoryInfo>
+		<PossibleProductions>
+			<ProductionChain name="Iron smelting" processingTime="4">
+				<Input>
+					<Item objectType="Raw Iron" amount="3" slotPosX="0" slotPosY="0" />
```

Comment:
`slotPosX` and `slotPosY` doesn't belong on each individual `Item`. It should probably be defined for the furniture as a whole.

Any particular reason it converts 3 for three? Is the idea that you can convert up to 48 iron for 48 steel? Or does it only take 3 items in the input?